index.js

```diff
@@ -1,3 +1,8 @@
 require('bare-process/global')
 
+const Worker = require('bare-worker', { with: { imports: './imports.json' } })
+const { resolve } = require('bare-path')
+
+Worker.preload(resolve('worker-preload'))
```

Comment:
```suggestion
Worker.preload(require.resolve('bare-process/global'))
```

Module specifiers must be resolved using `require.resolve()` to ensure that everything plays along with bundling.